com.gluon.scenebuilder.yml

```diff
@@ -0,0 +1,69 @@
+app-id: com.gluon.Scenebuilder
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: SceneBuilder
+finish-args:
+  - --require-version=0.10.3
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=xdg-config/gtk-3.0
+  - --filesystem=host
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=NPM_CONFIG_GLOBALCONFIG=/app/etc/npmrc
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --env=ZYPAK_SPAWN_LATEST_ON_REEXEC=0
+  - --filesystem=xdg-config/kdeglobals:ro
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+  - --system-talk-name=org.freedesktop.login1
+modules:
+  - name: cpio
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/cpio/cpio-2.12.tar.gz
+        sha256: 08a35e92deb3c85d269a0059a27d4140a9667a6369459299d08c17f713a92e73
+      - type: patch
+        path: cpio-2.13-mutiple-definition.patch
+
+  - name: rpm2cpio
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://github.com/ruda/rpm2cpio/archive/2017.tar.gz
+        sha256: d266a4f49e65b5d05956502549ceb1d5671f927375ff71299069e6d92363297a
+      - type: patch
+        path: rpm2cpio.patch
+    build-commands:
+      - install -D rpm2cpio.py /app/bin/rpm2cpio
+
+  - name: Scenebuilder
+    buildsystem: simple
+    build-commands:
+    - rpm2cpio SceneBuilder-18.0.0.rpm | cpio -idmv
+    - cp -r ./opt/scenebuilder/ -t /app/lib/
+    # - cp -r ./opt/scenebuilder/lib -t /app/lib/
+    # - ls -l 
+    - install com.gluon.scenebuilder.metainfo.xml /app/share/appdata/
+    - install -Dm755 apply_extra.sh /app/bin/SceneBuilder
+    sources:
+    - type: file
+      filename: SceneBuilder.rpm
+      url: https://download2.gluonhq.com/scenebuilder/18.0.0/install/linux/SceneBuilder-18.0.0.rpm
```

Comment:
so this is binary. x86_64 Only? you ar build an aarch64 flatpak as well.

Reply:
I have added
only-arches:
    - x86_64
but still not fix the issue

Reply:
that's not enough. See the documentation
https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

However it is disappointing.

Reply:
Thanks